<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="5CD138MBF9" ignored="0" name="Edge Test" tests="1" failures="1" timestamp="2021-12-03T09:17:18 IST" time="34.119" errors="0">
  <testcase name="loginTest" time="18.572" classname="org.cbsl.testcases.TC_AddEmployee">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.cbsl.testcases.TC_AddEmployee.loginTest(TC_AddEmployee.java:36)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Edge Test -->
